nix/nix-test-runner.nix

```diff
@@ -2,10 +2,16 @@ let
   flakeLock = builtins.fromJSON (builtins.readFile ../flake.lock);
   src = builtins.fetchTree flakeLock.nodes.nix-test-runner.locked;
 in
-{ pkgs ? import ./nixpkgs.nix { }
-  # Use last pinned crate2nix packages to build the test runner
+{ system
+, pkgs ? import
+    "${builtins.fetchTree (builtins.getAttr flakeLock.nodes.crate2nix_stable.inputs.nixpkgs flakeLock.nodes)
+      .locked}"
+    { inherit system; }
```

Comment:
This is the first point where I noticed the new nixpkgs causing problems. It wasn't enough to use the stable crate2nix - it had to be the stable crate2nix _with the same nixpkgs_ to get this working.